web/docs/installation/README.md

```diff
@@ -77,12 +77,14 @@ Copy the hidden files:
 
 ::: tip IMPORTANT
 
-This command should only be used once. The new `.env` is for the Firebase Functions API and the `.env.server` is for the Docker based Cloud Run Servers. Both files are for local development only and likely do not require any changes, except for the project ID that you created above.
-
-Also, make sure to update the `.firebaserc` with the project ID after this command is executed, as well.
+The following command should only be used once.
 
     npm run copy
 
+The new `.env` is for the Firebase Functions API and the `.env.server` is for the Docker based Cloud Run Servers. Both files are for local development only and likely do not require any changes, except for the project ID that you created above.
```

Comment:
It's unclear to me if I need to change anything in the `.env` or `.env.server` files here.  I'm going with no changes, but see that in `.env` and `.env.server` I should possibly be updating the value for `GOOGLE_CLOUD_PROJECT` to my Firebase project ID.  @derekherman which is correct?

Reply:
@jeffpaul Yes, you would update to your project ID. Good catch.

Reply:
`GOOGLE_CLOUD_PROJECT` is the same as the Firebase project ID they are one in the same.

Reply:
@jeffpaul did you want to make any changes to this knowing what you know now?